blogposts/2022/sourcegraph-non-devs.md

```diff
@@ -0,0 +1,207 @@
+
+**Follow-up situation:** What anchor text do we generally use for the email link, or do we display the direct email? [Search](https://sourcegraph.com/search?q=context:global+r:sourcegraph/+mailto:feedback%40sourcegraph.com&patternType=literal):
+
+<p>
+    <iframe src="https://sourcegraph.com/embed/notebooks/Tm90ZWJvb2s6MTg0?theme=light" frameborder="0" sandbox="allow-scripts allow-same-origin allow-popups" style="width: 100%; height: 175px;"></iframe>
+</p>
+
+### Should we use “prototype” or “experimental” for early-stage product features?
+
+[Feature classifications are important](https://handbook.sourcegraph.com/departments/product-engineering/product/process/beta_and_experimental_feature_labels#why-we-use-these-labels) because they help us set user expectations, classify support SLAs and ownership, and communicate internally about the maturity of a feature. In choosing what to call our pre-beta feature level, we wanted to quickly see what teammates had already gravitated towards.
+
+**Searches:**
+
+[Search one](https://sourcegraph.com/search?q=context:global+r:sourcegraph/sourcegraph%24+experimental+count:all&patternType=regexp)
+
+<p>
+    <iframe src="https://sourcegraph.com/embed/notebooks/Tm90ZWJvb2s6MTg1?theme=light" frameborder="0" sandbox="allow-scripts allow-same-origin allow-popups" style="width: 100%; height: 175px;"></iframe>
+</p>
+
+[Search two](https://sourcegraph.com/search?q=context:global+r:sourcegraph/sourcegraph%24+prototype+count:all&patternType=regexp)
+
+<p>
+    <iframe src="https://sourcegraph.com/embed/notebooks/Tm90ZWJvb2s6MTg2?theme=light" frameborder="0" sandbox="allow-scripts allow-same-origin allow-popups" style="width: 100%; height: 175px;"></iframe>
+</p>
+
+**Impact:** In 60 seconds, we determined we used “experimental” fives times as often as “prototype.” Given that already existing, informal tendency, we [codified “experimental” in our handbook](https://handbook.sourcegraph.com/departments/product-engineering/product/process/beta_and_experimental_feature_labels) as the official Sourcegraph label.
+
+## Analytics and Business Operations
+
+_This section was written by a member of the Business Operations team. He can give other teammates demos of Sourcegraph, but they’re generally prefaced with “Joel can do this better than me.”_
+
+### How has the Sourcegraph product data architecture changed over time?
+
+The architecture of our internally-facing data is ever-changing, so it’s critical we’re able to keep up with it so our data models are up-to-date and accurate.
+
+**Searches:**
+
+[This search](https://sourcegraph.com/search?q=context:%40ericbm+repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+file:%5Einternal/database/schema%5C.md+type:diff&patternType=literal) shows how our Postgres db structure changes over time.
```

Comment:
```suggestion
[This search](https://sourcegraph.com/search?q=context:%40ericbm+repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+file:%5Einternal/database/schema%5C.md+type:diff&patternType=literal) shows how our PostgreSQL database structure changes over time.
```
Making consistent with https://handbook.sourcegraph.com/company-info-and-process/communication/content_guidelines/terminology_guidelines/#term-usage. Also "db" feels a little informal 😅